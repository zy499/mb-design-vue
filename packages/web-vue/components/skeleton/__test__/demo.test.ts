import demoTest from '../../../scripts/demo-test';

demoTest('skeleton');
