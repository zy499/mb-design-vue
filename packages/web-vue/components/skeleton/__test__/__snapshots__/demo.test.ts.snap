// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<skeleton> demo: render [animation] correctly 1`] = `
"<div class=\\"mb-space mb-space-vertical\\" style=\\"width: 100%;\\">
  <div class=\\"mb-space-item\\" style=\\"margin-bottom: 24px;\\">
    <div class=\\"mb-space mb-space-horizontal mb-space-align-center\\">
      <div class=\\"mb-space-item\\" style=\\"margin-right: 8px;\\"><span>Animation</span></div>
      <div class=\\"mb-space-item\\"><button type=\\"button\\" role=\\"switch\\" aria-checked=\\"true\\" class=\\"mb-switch mb-switch-type-circle mb-switch-checked\\"><span class=\\"mb-switch-handle\\"><span class=\\"mb-switch-handle-icon\\"></span></span><!--  prettier-ignore  -->
          <!--v-if-->
        </button></div>
    </div>
  </div>
  <div class=\\"mb-space-item\\">
    <div class=\\"mb-skeleton mb-skeleton-animation\\">
      <div class=\\"mb-space mb-space-vertical\\" style=\\"width: 100%;\\">
        <div class=\\"mb-space-item\\" style=\\"margin-bottom: 24px;\\">
          <ul class=\\"mb-skeleton-line\\">
            <li class=\\"mb-skeleton-line-row\\" style=\\"height: 20px;\\"></li>
          </ul>
          <ul class=\\"mb-skeleton-line\\">
            <li class=\\"mb-skeleton-line-row\\" style=\\"height: 20px; margin-top: 15px;\\"></li>
          </ul>
          <ul class=\\"mb-skeleton-line\\">
            <li class=\\"mb-skeleton-line-row\\" style=\\"height: 20px; margin-top: 15px;\\"></li>
          </ul>
        </div>
        <div class=\\"mb-space-item\\">
          <div class=\\"mb-skeleton-shape mb-skeleton-shape-square mb-skeleton-shape-medium\\"></div>
        </div>
      </div>
    </div>
  </div>
</div>"
`;

exports[`<skeleton> demo: render [basic] correctly 1`] = `
"<div class=\\"mb-skeleton\\">
  <div class=\\"mb-space mb-space-vertical\\" style=\\"width: 100%;\\">
    <div class=\\"mb-space-item\\" style=\\"margin-bottom: 24px;\\">
      <ul class=\\"mb-skeleton-line\\">
        <li class=\\"mb-skeleton-line-row\\" style=\\"height: 20px;\\"></li>
      </ul>
      <ul class=\\"mb-skeleton-line\\">
        <li class=\\"mb-skeleton-line-row\\" style=\\"height: 20px; margin-top: 15px;\\"></li>
      </ul>
      <ul class=\\"mb-skeleton-line\\">
        <li class=\\"mb-skeleton-line-row\\" style=\\"height: 20px; margin-top: 15px;\\"></li>
      </ul>
    </div>
    <div class=\\"mb-space-item\\">
      <div class=\\"mb-skeleton-shape mb-skeleton-shape-square mb-skeleton-shape-medium\\"></div>
    </div>
  </div>
</div>"
`;

exports[`<skeleton> demo: render [type] correctly 1`] = `
"<div class=\\"mb-skeleton\\">
  <div class=\\"mb-space mb-space-horizontal mb-space-align-center\\">
    <div class=\\"mb-space-item\\" style=\\"margin-right: 24px;\\">
      <div class=\\"mb-skeleton-shape mb-skeleton-shape-square mb-skeleton-shape-small\\"></div>
    </div>
    <div class=\\"mb-space-item\\" style=\\"margin-right: 24px;\\">
      <div class=\\"mb-skeleton-shape mb-skeleton-shape-square mb-skeleton-shape-medium\\"></div>
    </div>
    <div class=\\"mb-space-item\\" style=\\"margin-right: 24px;\\">
      <div class=\\"mb-skeleton-shape mb-skeleton-shape-square mb-skeleton-shape-large\\"></div>
    </div>
    <div class=\\"mb-space-item\\" style=\\"margin-right: 24px;\\">
      <div class=\\"mb-skeleton-shape mb-skeleton-shape-circle mb-skeleton-shape-small\\"></div>
    </div>
    <div class=\\"mb-space-item\\" style=\\"margin-right: 24px;\\">
      <div class=\\"mb-skeleton-shape mb-skeleton-shape-circle mb-skeleton-shape-medium\\"></div>
    </div>
    <div class=\\"mb-space-item\\">
      <div class=\\"mb-skeleton-shape mb-skeleton-shape-circle mb-skeleton-shape-large\\"></div>
    </div>
  </div>
</div>"
`;
